PixArt/models/PixArtMS.py

```diff
@@ -224,15 +224,15 @@ def forward(self, x, timesteps, context, img_hw=None, aspect_ratio=None, **kwarg
                 device=x.device
             ).repeat(bs, 1)
         else:
-            data_info["img_hw"] = img_hw.to(x.dtype).to(x.device)
+            data_info["img_hw"] = img_hw.to(dtype=x.dtype, device=x.device)

```

Comment:
Isn't this technically slower? Guess it probably doesn't matter with how small these tensors are lol.